deb/build

```diff
@@ -44,15 +69,29 @@ fi
 libversion=`echo $version_number | grep -o -P -e "([0-9]+\.?)+$"`
 lib_major=$(echo $libversion | cut -f1 -d .)
 
+# create temp file to write changelog to
 changelog=`mktemp`
 trap "rm -f '$changelog'; exit 1" INT TERM QUIT
 
-so_libname="./install/usr/lib/$so_filename"
+# (re)create symbolic links with the distributed package name, dist_pkgname
+cd $libdir
+touch $dist_so_libname.$libversion
+ln -sf $dist_so_libname.$libversion $dist_so_libname.$lib_major
+ln -sf $dist_so_libname.$lib_major $dist_so_libname
```

Comment:
No the api level is appended right after the pkgname, ie. `api_level=6` would generate the package  `libdrizzle-redux6`